docs/source/aryn_cloud/accessing_the_partitioning_service.rst

```diff
@@ -107,4 +126,16 @@ Your results have been saved to ``document.json``.
 
 .. code:: bash
 
-    cat document.json
\ No newline at end of file
+    cat document.json
+
+Different Formats
++++++++++++++++++
+
+.. code:: bash
+
+    export ARYN_API_KEY="PUT API KEY HERE"
+    curl -s -N -D headers "https://api.aryn.cloud/v1/document/partition" -H "Authorization: Bearer $ARYN_API_KEY" -F "pdf=@document.pdf" | tee document.json
+    curl -s -N -D headers "https://api.aryn.cloud/v1/document/partition" -H "Authorization: Bearer $ARYN_API_KEY" -F "pdf=@document.docx" | tee document.json
```

Comment:
"pdf=@..."  this is confusing for non PDFs. Can we change this?

Reply:
Yes. I will update this this week. @bsowell recommends we do not block the launch on this, unless you veto.

Reply:
Let's not block, but this should get updated in the next couple days if we don't block.